vector-im/element-android
# Live Location Sharing - User List Bottom Sheet [PSF-890]

## Type of change


## Content

This PR implements a bottom sheet on maximized live location sharing screen. The bottom sheet contains the user list who currently share their live location. If the current user is sharing a live location the list item also has a button to allow stopping location sharing. Clicking a user should zoom to user's location.

## Motivation and context

Closes #6012 

## Screenshots / GIFs

https://user-images.githubusercontent.com/1254401/170721995-c352d311-7095-45c4-83b7-763411ee4dce.mp4 

https://user-images.githubusercontent.com/1254401/170722362-dbbea1d2-415f-4497-98f5-f01798f49c35.mp4


## Tests

- Select location attachment icon from timeline composer
- Select "Share live location" in location screen
- Ideally share live location from another user in the same room too
- Live location item should be visible in timeline, tap on the item, should navigate to maximized map screen
- You should see all users' avatars on map and the bottom sheet with the user list
- You should see a stop sharing button next to your user's item in bottom sheet
- Selecting stop sharing button should remove your user on map and in bottom sheet list
- Selecting a user in bottom sheet should zoom to the selected user's location on map
- Bottom sheet should be extendable and scrollable

## Tested devices

- OS version(s):

## Checklist
